usagov/vote-gov
# [Vote 432] cypress test updates for state links and home page

link to ticket -> https://bixal-projects.atlassian.net/browse/VOTE-432

This PR will cover adding back the home page tests
updates that were made
- the test will check the accordion that is clickable and that text is visible to ensure that the function is working. 
- the 2nd part is checking all links that they have the right text in the hyperlink and that the href is current (the same format that we have for the footer test) to keep the constantly with other formats in the project 

adding state links test 

- This has been updated to match the same format that we are doing in the home page and footer links. since we can't control the status of the external site this will verify that the text is correct and that the href is correct as well.  The status of external sites is taken care of in a different test 

to test 
- pull down the branch 
- run `npm run start` spilt terminal 
- run `npm run cy:open` this will open the cypress runner and then run the `home-page.cy.js` and `state-links.cy.js` tests to verify they are working as expected
